packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -798,9 +823,10 @@ export function TransferPanelMain({
   )
 
   return (
-    <div className="flex flex-col pb-6">
+    <div className="flex flex-col pb-6 lg:space-y-1">
```

Comment:
i don't really like space because it uses margin in flex
if possible we should use gap